server/models/Notification.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const { Schema, ObjectId } = mongoose;
+
+const notificationSchema = new Schema({
+  recipient: { type: ObjectId, required: true },
```

Comment:
you'll want to index this field since you will be fetching by it a lot

Reply:
added indexing for the recipient field